Phat-Taupe-Pests/BangazonAPI
# Made Order Controller and Model, Post/Put/GetAll/GetSingle/Delete

## DESCRIPTION
Wrote the Order Controller and Model. Post/Put/GetAll/GetSingle/Delete verbs should all work.

## STEPS TO TEST
1. Clone this branch to a local repo
2. In terminal, run `dotnet ef update database` `dotnet build` then `dotnet run`
3. Test functionality
     * POSTMAN make sure that header key is 'content-type', value as 'application/json'
     * GET all, url http://localhost:5000/Order
     * GET single, url http://localhost:5000/Order/{id}
     * POST url http://localhost:5000/Order
```{ "customerID": {whatever ID you want} }```
     * PUT url http://localhost:5000/Order/{id}, make sure you do the entire object
```{"orderID": 2,"dateCreated": "2017-07-25T00:00:00", "customerID": 2,"customer": null}```
     * DELETE url http://localhost:5000/Order/{id}

## REFERENCE
[Ticket](https://github.com/Phat-Taupe-Pests/BangazonAPI/issues/4)
